javascript/javascript3/week2/game.js

```diff
@@ -0,0 +1,42 @@
+const apiEndpoint = 'https://api.exchangerate-api.com/v4/latest/EUR';
+let exchangeRates, currencies;
+
+function convert() {
+  const amount = document.getElementById('amount').value;
+  const from = document.getElementById('from').value;
+  const to = document.getElementById('to').value;
+  const result = amount * exchangeRates[to] / exchangeRates[from];
+  document.getElementById('result').innerHTML = `${amount} ${from} = ${result} ${to}`;
+}
+
+function populateCurrencies() {
+  currencies = Object.keys(exchangeRates);
+  const fromSelect = document.getElementById('from');
+  const toSelect = document.getElementById('to');
+  currencies.forEach(currency => {
```

Comment:
In order to have a clearer code, it is add a blank line between the different parts or sections of your function body.